packages/manager/apps/zimbra/public/translations/domains/diagnostic/Messages_fr_FR.json

```diff
@@ -0,0 +1,35 @@
+{
+  "zimbra_domain_modal_diagnostic_guide": "guide",
+  "zimbra_domain_modal_diagnostic_srv_title": "Diagnostique SRV",
+  "zimbra_domain_modal_diagnostic_srv_content_header": "Pour faciliter la configuration des boites mails sur vos terminaux, veuillez ajouter cette configuration à votre DNS (consulter notre <guide/>)",
+  "zimbra_domain_modal_diagnostic_srv_content_header_ovh_hosted_domain": "La configuration SRV permet la configuration automatique de vos boites mails sur vos clients de messagerie.<br/>Votre domaine est hébergé chez OVH Cloud, Nous allons procéder à la configuration de votre champ SRV.",
```

Comment:
**nitpick:** Instead of adding HTML in translation, prefer splitting in 2 sentences.

Reply:
To me splitting in multiple sentences would increase complexity on the component side, since sometimes it will be multiple sentence and sometimes only one, it gives also control to the content in the translation file which is not a bad thing to me (less dev)